tests/test_dir_layouts.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+from pathlib import Path
+import shutil
+import bids
```

Comment:
L1 and L4 can be deleted as these imports are not being used

Reply:
I think pytest might be as I use a fixture from it, and bids will be used to validate that the outputs look correct.